safe_transaction_service/notifications/tasks.py

```diff
@@ -115,8 +90,8 @@ def send_notification_task(
         return 0, 0
 
     # Make sure notification has not been sent before
-    duplicate_notification = DuplicateNotification(address, payload)
-    if duplicate_notification.is_duplicated():
+    if not mark_notification_as_processed(address, payload):
+        # Notification was processed already
```

Comment:
In the line https://github.com/safe-global/safe-transaction-service/pull/1483/files#diff-13a79df51edabd3134f56efff3cb2876ed07f122ccbca0a755be8d919cf32f2cR87 we are sending notification, could be this notification duplicated?

Reply:
You are right, we should check it as soon as posible, updating that, done